transform/mattermost-analytics/models/marts/sales/_sales__models.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+
+
+models:
+  - name: fct_subscription_history
+    description: |
+      Cloud subscription seat history data.
+    columns:
+    - name: subscription_history_id
```

Comment:
It's a common practice to create a unique key for each row in fact tables. If the key is a combination of columns, then a surrogate key can be generated from these columns:

Here's an example: https://github.com/mattermost/mattermost-data-warehouse/blob/1a8f222effc50d1131f3377649da85c3fd473d35/transform/mattermost-analytics/models/marts/product/fct_active_users.sql#L16.

Reply:
The `subscription_history_id` field is a unique ID assigned to a subscription history event by the CWS - it's an alphanumeric 26 character string. I don't think we need a surrogate key given this?